advanced/pihole.cron

```diff
@@ -28,3 +28,6 @@
 00 00   * * *   root    PATH="$PATH:/usr/local/bin/" pihole flush once quiet
 
 @reboot root /usr/sbin/logrotate /etc/pihole/logrotate
+
+# Pi-hole: Grab remote version and branch every 10 minutes
+*/10 *  * * *   root    PATH="$PATH:/usr/local/bin/" pihole updatechecker
```

Comment:
This might be better run every 30 minutes, or even once a day. Though, I can see this being subjective and up to the user.

Reply:
Six times an hour seems to be fair. The reason for this frequency is not to check for updates very often, but to detect the branch the user is currently on (the determination of the current hash is also outsourced into `cron` as well)